test/core/Leaf.t.sol

```diff
@@ -286,12 +286,18 @@ contract TickTest is Test {
     Field level1 = Field.wrap(bound(_level1, 1, uint(LEVEL1_SIZE) - 1));
     Field level2 = Field.wrap(bound(_level2, 1, uint(LEVEL2_SIZE) - 1));
     Field level3 = Field.wrap(bound(_level3, 1, uint(LEVEL3_SIZE) - 1));
-    Tick tick = TickLib.tickFromBranch(tickPosInLeaf, level0, level1, level2, level3);
+    MgvStructs.LocalPacked local;
+    local = local.tickPosInLeaf(tickPosInLeaf);
+    local = local.level0(level0);
+    local = local.level1(level1);
+    local = local.level2(level2);
+    local = local.level3(level3);
+    Tick tick = TickLib.bestTickFromLocal(local);
```

Comment:
The test name still refers to the old `tickFromBranch` name

Reply:
fixed in adad68d184b477bc2c72959e208ac17ad791baa5

Reply:
I think that commit changes too much 😅 Adding `best` is IMO only needed and relevant when the arguments could identify multiple ticks; When the arguments determine one tick, then I don't think `best` is relevant and actually somewhat misleading since it indicates we choose between multiple possible ticks.

Reply:
fixed with revert + proper commit

Reply:
I'm still seeing this in TestTickTree?

`function bestTickFromBranch(uint posInLevel3, uint posInLevel2, uint posInLevel1, uint posInLevel0, uint posInLeaf)